src/App.jsx

```diff
@@ -1,6 +1,5 @@
 import React from "react";
-import { 
-  BrowserRouter as Router,
+import {
```

Comment:
In the future try to avoid changes that are out of scope

Reply:
hmm we deleted something we don't use in this file

Reply:
When should something like this be deleted or modified?  I thought if you find a mistake you can fix it there and then, do you need to go back and change branches to do this?